doc/policy/mempool-design.md

```diff
@@ -0,0 +1,105 @@
+# Mempool design and limits
+
+## Definitions
+
+We view the unconfirmed transactions in the mempool as a directed graph,
+with an edge from transaction B to transaction A if B spends an output created
+by A (ie, B is a **child** of A, and A is a **parent** of B).
```

Comment:
In commit "doc: Add design notes for cluster mempool and explain new mempool limits"

Writing nit: `ie,` -> `i.e.,`